KamerConnect/Models/ChatMessage.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace KamerConnect.Models;
+
+public class ChatMessage
+{
+    public Guid Id { get; set; }
+    public Guid ChatId { get; set; }
+    public Guid SenderId { get; set; }
+    public string Message { get; set; }
+    public DateTime SendAt { get; set; }
+
+    public ChatMessage(Guid id, Guid chatId, Guid senderId, string message, DateTime sendAt)
+    {
+        Id = id;
+        ChatId = chatId;
```

Comment:
Als het goed is heeft een chat de Messages erin zitten dan is er geen chatId nodig, want die zit dan al in het Parent object.